packages/unified/src/unified.ts

```diff
@@ -0,0 +1,110 @@
+import { AmplitudeBrowser } from '@amplitude/analytics-browser';
+import {
+  AmplitudeSessionReplay,
+  SessionReplayOptions,
+  sessionReplayPlugin,
+  SessionReplayPlugin,
+} from '@amplitude/plugin-session-replay-browser';
+import {
+  IExperimentClient,
+  ExperimentConfig,
+  ExperimentPlugin,
+  experimentPlugin,
+} from '@amplitude/plugin-experiment-browser';
+import { BrowserClient, BrowserOptions, LogLevel } from '@amplitude/analytics-core';
+
+export interface UnifiedSharedOptions {
+  serverZone?: 'US' | 'EU';
+  instanceName?: string;
+  /**
+   * If logLevel is debug:
```

Comment:
Do these debug modes just enable additional logging? If not, I'm hesitant to have them controlled by log level. There's no equivalent to debug mode on the mobile sdks that I'm aware of, but maybe we make `debug` a top level property, at least on ts for now?

Reply:
experiment's debug seems to be just logs https://github.com/amplitude/experiment-js-client/blob/29b1640f54ac0cab1ac1795d0ffd104a1fbf44d5/packages/experiment-browser/src/config.ts#L13-L17. But `sr.debugMode` adds additionally event properties https://github.com/amplitude/Amplitude-TypeScript/blob/7a665d55fff89092ed5f2bb94caa1eb2c7efe5b1/packages/session-replay-browser/src/config/types.ts#L81-L86. 

How about not setting `sr.config.debugMode` here and keep the rest?
``` 
  /**
   * If logLevel is debug:
   * - analytics.config.logLevel = debug
   * - sr.config.logLevel = debug
   * - sr.config.debugMode = true // Remove this
   * - experiment.config.debug = true
   */
```

Reply:
Both of these flags warn not to turn them on in prod environments, so I don't think setting them implicitly from log level (which should be fine to set to any level in prod environments) is a good idea.

Reply:
Thanks @crleona, I remove logLevel at https://github.com/amplitude/Amplitude-TypeScript/pull/1039/commits/b0f3ea8a18273edc07f4cc023746fc1f89924b12